migrations/sql/V2023.09.08.15.10__add_verifiable_credential_connection_table.sql.sql

```diff
@@ -0,0 +1,15 @@
+
+CREATE TABLE mine_verifiable_credential_connection
```

Comment:
probably want to fix .sql.sql. Validation performed on these file names gets tripped pretty easily.

Reply:
FIXED. yes flyway can be fickle